src/cli/cli_network_data.cpp

```diff
@@ -802,7 +827,9 @@ otError NetworkData::OutputBinary(bool aLocal)
  * 6LoWPAN Context IDs are listed under `Contexts` header:
  * * The prefix
  * * Context ID
- * * Compress flag (`c` if marked or `-` otherwise).
+ * * Flags
+ *   * s: Stable flag
+ *   * c: Compress flag
```

Comment:
I would add - to the list for completeness, if that's what prints when it's neither s nor c

Reply:
Good idea. Added `-` to the list.